packages/e2e/lib/common.dart

```diff
@@ -0,0 +1,57 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+
+/// An object sent from e2e back to the Flutter Driver in response to
+/// `request_data` command.
+class Response {
+  String _failureDetails;
+
+  final bool _allTestsPassed;
+
+  /// Constructor which receives tests results' flag and failure details.
```

Comment:
I'm not sure the word "flag" describes "allTestsPassed" very well.

Reply:
done.